src/patternfly/components/CodeEditor/examples/CodeEditor.css

```
@@ -0,0 +1,3 @@
+#ws-core-c-code-editor-with-full-height-modifiers .ws-preview-html {
```

If you update the example name, we'll need to update this id

```suggestion
#ws-core-c-code-editor-with-full-height-modifier .ws-preview-html {
```